### PR TITLE
Add compat data for BluetoothRemoteGATTService

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTService

### CODE REVIEW

File: api/BluetoothRemoteGATTService.json [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+              "version_removed": "53",
+              "notes": "Behind a preference flag in Chrome OS."
+            },
+            {
+              "version_added": "51",
+              "version_removed": "53",
+              "notes": "Linux support."
+            },
+            {
+              "version_added": "53",
```
Comment (connorshea) [inline api/BluetoothRemoteGATTService.json]: Newest supported version should be at the top, not the bottom. This is true across the file.

File: api/BluetoothRemoteGATTService.json [hunk 2]
```diff
@@ -9,29 +9,29 @@
           },
           "chrome": [
             {
-              "version_added": "45",
-              "version_removed": "53",
-              "notes": "Behind a preference flag in Chrome OS."
+              "version_added": "53",
+              "notes": "Mac support added and <a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
```
Comment (haykam821) [inline api/BluetoothRemoteGATTService.json]: Invalid URL.

File: api/BluetoothRemoteGATTService.json [hunk 3]
```diff
@@ -9,29 +9,29 @@
...
-              "notes": "Mac support added and <a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
+              "version_added": "45",
+              "version_removed": "53",
+              "notes": "Behind a preference flag in Chrome OS."
             }
           ],
           "chrome_android": [
+            {
+              "version_added": "53",
+              "notes": "<a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
```
Comment (haykam821) [inline api/BluetoothRemoteGATTService.json]: Invalid URL.

File: api/BluetoothRemoteGATTService.json [hunk 4]
```diff
@@ -285,29 +285,29 @@
             },
             "chrome": [
               {
-                "version_added": "45",
-                "version_removed": "53",
-                "notes": "Behind a preference flag in Chrome OS."
+                "version_added": "53",
+                "notes": "Mac support added and <a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
```
Comment (haykam821) [inline api/BluetoothRemoteGATTService.json]: Invalid URL.

File: api/BluetoothRemoteGATTService.json [hunk 5]
```diff
@@ -285,29 +285,29 @@
...
-                "notes": "Mac support added and <a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
+                "version_added": "45",
+                "version_removed": "53",
+                "notes": "Behind a preference flag in Chrome OS."
               }
             ],
             "chrome_android": [
+              {
+                "version_added": "53",
+                "notes": "<a href='https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md'>Origin Trial<>."
```
Comment (haykam821) [inline api/BluetoothRemoteGATTService.json]: Invalid URL.

**Pull request merged**